native/run

```diff
@@ -29,4 +29,4 @@ program
     execSync(`yarn cross-env ${buildConfig} yarn react-native run-android --no-packager --appId ${applicationId} ${productionFlag}`, { stdio: 'inherit' })
   })
 
-program.parse(process.argv)
+program.parse(process.argv)
```

Comment:
Revert

Reply:
I just copied the old content into the file again. This was easier for me

Reply:
Fine of course, I just meant restore the old state :)